util/config/config.go

```diff
@@ -37,9 +37,9 @@ const (
 	InitialIssueAmount           = 700000000 * common.StorageFactor
 	TotalMiningRewards           = 300000000 * common.StorageFactor
 	TotalRewardDuration          = uint32(25)
-	InitialReward                = 18000000 * common.StorageFactor
+	InitialReward                = common.Fixed64(18000000 * common.StorageFactor)
```

Comment:
Should we use
common.Fixed64(18000000 * common.StorageFactor)
or
common.Fixed64(18000000) * common.Fixed64(common.StorageFactor)
?